sanic/reloader_helpers.py

```diff
@@ -1,9 +1,10 @@
 import os
-import sys
 import signal
 import subprocess
-from time import sleep
+import sys
+
 from multiprocessing import Process
+from time import sleep
```

Comment:
weird, maybe we add these to `known_standard_library`

Reply:
Perhaps a "leaner" `isort` config,  so it won't add a blank line between `import ...` and `from ... import ...` ?

Reply:
I'm fine with one blank between different import types. no blank is fine too for me.